tools/clang/tools/dxcompiler/dxcontainerbuilder.cpp

```diff
@@ -159,7 +159,7 @@ HRESULT STDMETHODCALLTYPE DxcContainerBuilder::SerializeContainer(_Out_ IDxcOper
     HRESULT valHR = S_OK;
     if (m_RequireValidation) {
       CComPtr<IDxcValidator> pValidator;
-      IFT(CreateDxcValidator(IID_PPV_ARGS(&pValidator)));
+      IFT(CreateDxcValidator(__uuidof(IDxcValidator), (LPVOID*)&pValidator));
```

Comment:
Out of curiosity, was there a reason to exclude IID_PPV_ARGS except that it's a bit silly and not terribly helpful?

Reply:
I believe on Windows under the hood, IID_PPV_ARGS is defined as
```cpp
#define IID_PPV_ARGS(x) __uuidof(x), (LPVOID*)x
```
The reason I had got rid of IID_PPV_ARGS was that it takes a __variable__ (pointer) rather than a __Type/Class__ name. And we can't resolve the ``__uuidof(variable)``

But, I just remembered that we *can* use an object to call the static method of its class.

So if we define
```cpp
#define IID_PPV_ARGS(x) x.uuidof(), (LPVOID*)x
```
that should work. I'll give it a try and provide an update.
Thanks